std/flags/mod.ts

```diff
@@ -15,7 +15,7 @@ export interface ArgParsingOptions {
    * the result `['--']` with everything after the `--`. Here's an example:
    *
    *      // $ deno run example.ts -- a arg1
-   *      import { parse } from "https://deno.land/std/flags/mod.ts";
+   *      import { parse } from "https://deno.land/std@0.70.0/flags/mod.ts";
```

Comment:
Imports from docstrings should be removed - they will get outdated with each new release

Reply:
done